plenum/test/node_request/test_propagate/test_node_request_only_needed_propagates.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+from plenum.server.quorums import Quorum
+from plenum.test.helper import sdk_send_random_and_check
+
+from plenum.test.malicious_behaviors_node import dont_send_messages_to, dont_send_propagate
+from plenum.test.spy_helpers import get_count
+
+
+@pytest.fixture(scope='function')
+def setup(txnPoolNodeSet):
+    faulty_node = txnPoolNodeSet[-1]
+    # Set quorum a bit more, so that faulty_node will request propagates
+    faulty_node.quorums.propagate = Quorum(3)
```

Comment:
3 == n-f, isn't it? So why do we need to override the quorum value here?

Reply:
I set it to 3 because propagate quorum here is 2 (f+1). So if we will let alpha to send us propagate, we will collect quorum (including ourself) and will not request propagates.